lib/lightning/workflows.ex

```diff
@@ -253,11 +253,23 @@ defmodule Lightning.Workflows do
       with {:ok, _} <- result do
         workflow
         |> Repo.preload([:triggers], force: true)
+        |> notify_of_affected_kafka_triggers()
         |> Events.workflow_updated()
       end
     end)
   end
 
+  defp notify_of_affected_kafka_triggers(workflow) do
+    workflow
+    |> tap(fn %{triggers: triggers} ->
```

Comment:
Nice awereness to avoid colateral effects using tap @rorymckinley. I would just surface it to the calling point to make it evident from the beginning that this is call doesn't affect the result of the workflow pipe on line 256.
```
|> Repo.preload([:triggers], force: true)
|> tap(&notify_kafka_triggers/1)
```
where
```
defp notify_kafka_triggers(%{triggers: trigger}) do
```

Reply:
Obrigado @jyeshe - I made the change.